airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.tsx

```diff
@@ -0,0 +1,181 @@
+  );
+};
+
+const DbtJobsList = ({
+  jobs,
+  remove,
+  isValid,
+  dirty,
+}: {
+  jobs: DbtCloudJob[];
+  remove: (i: number) => void;
+  isValid: boolean;
+  dirty: boolean;
+}) => (
+  <div className={classNames(styles.jobListContainer, styles.emptyListContent)}>
+    <p className={styles.contextExplanation}>After an Airbyte sync job has completed, the following jobs will run</p>
+    {jobs.length ? (
+      jobs.map((_j, i) => <JobsListItem key={i} jobIndex={i} removeJob={() => remove(i)} />)
+    ) : (
+      <>
+        <img src="/images/octavia/worker.png" alt="An octopus wearing a hard hat, tools at the ready" />
+        No transformations
+      </>
+    )}
+    <div className={styles.jobListButtonGroup}>
+      <Button className={styles.jobListButton} type="reset" variant="secondary">
+        Cancel
+      </Button>
+      <Button className={styles.jobListButton} type="submit" variant="primary" disabled={!dirty || !isValid}>
+        Save changes
+      </Button>
+    </div>
+  </div>
+);
+
+// TODO give feedback on validation errors (red outline and validation message)
+const JobsListItem = ({ jobIndex, removeJob }: { jobIndex: number; removeJob: () => void }) => {
+  return (
+    <Card className={styles.jobListItem}>
+      <div className={styles.jobListItemIntegrationName}>
+        <img src="/images/external/dbt-bit_tm.png" alt="dbt logo" />
+        dbt Cloud transform
+      </div>
+      <div className={styles.jobListItemInputGroup}>
+        <div className={styles.jobListItemInput}>
+          <Field name={`jobs.${jobIndex}.account`}>
+            {({ field }: FieldProps<string>) => (
+              <>
+                <label htmlFor={`jobs.${jobIndex}.account`} className={styles.jobListItemInputLabel}>
+                  Account ID
+                </label>
+                <Input {...field} type="text" />
+              </>
+            )}
+          </Field>
+        </div>
+        <div className={styles.jobListItemInput}>
+          <Field name={`jobs.${jobIndex}.job`}>
+            {({ field }: FieldProps<string>) => (
+              <>
+                <label htmlFor={`jobs.${jobIndex}.job`} className={styles.jobListItemInputLabel}>
+                  Job ID
+                </label>
+                <Input {...field} type="text" />
+              </>
+            )}
+          </Field>
+        </div>
+        <button type="button" className={styles.jobListItemDelete} onClick={removeJob}>
+          <FontAwesomeIcon icon={faXmark} />
+        </button>
+      </div>
+    </Card>
+  );
+};
+
+const NoDbtIntegration = ({ className }: { className: string }) => {
+  const { workspaceId } = useCurrentWorkspace();
+  const dbtSettingsPath = `/${RoutePaths.Workspaces}/${workspaceId}/${RoutePaths.Settings}/dbt-cloud`;
+  return (
+    <div className={classNames(className, styles.emptyListContent)}>
+      <p className={styles.contextExplanation}>After an Airbyte sync job has completed, the following jobs will run</p>
```

Comment:
I'd remove this sentence in the integration not setup state, since it feels a bit weird telling, that the following transformations will run, while we know there are no transformations, and the user isn't setup yet. I'd maybe phrase it more like:

"To run dbt Cloud transformations when syncing data, setup your dbt Cloud account in the <link>settings</link>."